statchatui/src/Home/Home.jsx

```diff
@@ -1,22 +1,40 @@
 import React from "react";
 import Stats from "../Stats/Stats";
-
+const GAME_NAMES = {
+  APEX: "Apex",
+};
 const Home = ({ onStatsChange, stats }) => {
   const handleSubmit = async (event) => {
     event.preventDefault();
 
     const formData = {
-      user: event.target.user.value,
+      gamertag: event.target.gamertag.value,
       platform: event.target.platform.value,
+      gameName: GAME_NAMES.APEX,
     };
-    const user = formData.user;
+
+    try {
+      await fetch(`http://localhost:3002/users/profile`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(formData),
+        credentials: "include",
+        //this includes any cookies or credentials the server might need to validate the user, without this, the user will not be identified in the post request and will not update the correct user in the database
```

Comment:
Same, this should be moved to above the "credentials" line. 

This is correct but not the complete reason we want to have "include". The default value is "same-origin". The reason we have it set to "include" is to support local development (on localhost) because our client and API server are on different hosts.